src/openms/include/OpenMS/FILTERING/DATAREDUCTION/SplineInterpolatedPeaks.h

```diff
@@ -120,14 +125,14 @@ class OPENMS_DLLAPI SplineSpectrum
         * @brief returns spline interpolated intensity at m/z
         * (fast access since we can start search from lastPackage)
         */
-        double eval(double mz);
+        double eval(double pos);
 
         /**
         * @brief returns the next sensible m/z position
         *  for scanning through a spectrum
         * (fast access since we can start search from lastPackage)
```

Comment:
lastPackage is not part of the interface so it doesn't help much for the documentation.
not clear what this means